protocol/migrations/sqlite/1708085975_user_messages_remove.up.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE IF NOT EXISTS user_messages_remove (
```

Comment:
I wonder, instead of creating a new table, can't we just check the community description for the banned members. If the member is banned, we just delete/ignore the messages?